modules/books.js

```diff
@@ -4,14 +4,14 @@ const mongoose = require('mongoose');
 const { Schema } = mongoose;
 
 const bookSchema = new Schema({
-  Books:Array,
+  Books: Array,
 })
 
 const booksModel = mongoose.model('books', bookSchema);
```

Comment:
Feedback here: see task card 2: `Modularize your code by putting your schema and model in its own separate file and requiring them into your server.`